broadinstitute/pearl
# JN-550 sandbox safety

This puts surveys and site contents into read-only mode if a non-sandbox environment is selected.  It also enhances the service-layer safeguards so that now not even superusers can save to non-sandbox environments.  We can always update the DB directly, and this add even more misclick-safety

TO TEST:
1. go to `https://localhost:3000/ourhealth/studies/ourheart/env/sandbox/forms/preEnroll/oh_oh_preenroll`
2. make a small change in the survey
3. observe the save button is enabled
4. use the environment selector to go to 'irb' or 'live
5. confirm the survey is put into read-only mode
